### DISCUSSION TITLE
Someone knows if decostand renormalizes the data after the translation with the pseudocounts?

### DISCUSSION THREAD

**Discussion by sciencerabbit19** [vegandevs/vegan#669] [Q&A]

**Comment by gavinsimpson** [comment]
If all else fails, you could read the source code

**Comment by jarioksa** [comment]
`pseudocount` is only used in `decostand` methods `"clr"` and `"alr"`.  I have no idea what "renormalization" means, but the documents do not mention "renormalization", and I think this means that it is not applied (what ever it is).
